test/index.js

```diff
@@ -545,6 +545,28 @@ describe('def', function() {
                    'as its first argument; received ["abc"]'));
   });
 
+  it('supports "enum" types', function() {
+    var def = $.create($.env.concat([$.Enum, $.ValidDate, $.ValidNumber]));
```

Comment:
It seems to me we want this:

```javascript
//  TimeUnit :: Type
var TimeUnit = $.Enum(['ms', 'seconds']);

var def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
```

As an aside, it bothers me *slightly* that we have `("ms" | "seconds")` rather than `("milliseconds" | "seconds")` or `("ms" | "s")`. ;)